### ISSUE TITLE
Documentation is inaccessible

### ISSUE THREAD

**Issue by salixor** [nextcloud/nextcloud-vue#2820]
When trying to access https://nextcloud-vue-components.netlify.app/, an error is returned :

```
Error: Minified React error #321; visit https://reactjs.org/docs/error-decoder.html?invariant=321 for the full message or use the non-minified dev environment for full errors and additional helpful warnings.
Ribbon@webpack://@nextcloud/vue/./node_modules/react-styleguidist/lib/client/rsg-components/Ribbon/Ribbon.js?:12:107
div
StyleGuideRenderer@webpack://@nextcloud/vue/./node_modules/vue-styleguidist/lib/client/rsg-components/VsgStyleGuide/StyleGuideRenderer.js?:81:17
_class@webpack://@nextcloud/vue/./node_modules/react-styleguidist/lib/client/rsg-components/Styled/Styled.js?:63:28
StyleGuide@webpack://@nextcloud/vue/./node_modules/react-styleguidist/lib/client/rsg-components/StyleGuide/StyleGuide.js?:91:29

This may be due to an error in a component you are overriding, or a bug in React Styleguidist.

If you believe this is a bug, [please submit an issue](https://github.com/styleguidist/react-styleguidist/issues).
```

**Comment by jilleJr** [comment]
I checked out the recent commits and found that it seems to be caused by 8ea2e155e4e755b4178d781674d91aff016d5fc1 (#2802), as the styleguidist worked in its parent, 1f12fe9b2214e9d081affc8d2bb1b5076fd12f60

I personally really need these docs as I'm too newbie, and a workaround for me was to clone this repository and run:

```sh
git checkout 1f12fe9b2214e9d081affc8d2bb1b5076fd12f60
npm install
npm run styleguide
```

And then visit <https://localhost:6060/>

I've no idea how to solve the issue (without downgrading)

**Comment by salixor** [comment]
I'm closing this issue since the cause has been found and resolved in #2823
